plugins/draw-tools.js

```diff
@@ -596,8 +596,8 @@ window.plugin.drawTools.snapToPortals = function() {
         if (visibleBounds.contains(lls[i])) {
           testCount++;
           var newll = findClosestPortalLatLng(lls[i]);
```

Comment:
```suggestion
          var newll = findClosestPortalLatLng(lls[i]);
          // Using a hard comparison instead of a soft "equal" to fix "snap to portals" precision.
          // See 
https://github.com/IITC-CE/ingress-intel-total-conversion/pull/481
```